modules/op-specifying-manual-approval-task.adoc

```diff
@@ -41,6 +41,16 @@ spec:
       value: '2'
     - name: timeout
       value: '60m'
+status:
+  state: pending
+  approvers:
+  - user1
+  - user2
+  - user3
+  approvalsRequired: 2
+  approvalsReceived: 0
+  approversResponse: []
```

Comment:
better to show an example with approverresponse. could we take one example from approved and rejected from here - https://github.com/openshift-pipelines/manual-approval-gate/blob/main/docs/APPROVAL_TASK_GUIDE.md#status-fields-1

Reply:
Fixed.